src/foam/u2/tag/Select.js

```diff
@@ -38,7 +38,14 @@ foam.CLASS({
         return undefined;
       }
     },
-    'size'
+    'size',
+    {
+      name: 'dropDownHeader',
+      factory: function() {
+        return undefined;
+      },
+      documentation: 'drop down header is use to indicate the category of choices. it located above placeholder'
```

Comment:
But unlike RichChoiceView, there is only one "category", which means it isn't really a category. So this is just a "header". Could we fix the documentation and rename it to something like just 'header'?